src/gui/qjacktrip.cpp

```diff
@@ -948,6 +948,7 @@ void QJackTrip::start()
                     m_jackTrip->setOutputDevice(
                         m_ui->outputDeviceComboBox->currentText().toStdString());
                 }
+                AudioInterface::setPipewireLatency(bufferSize, sampleRate);
```

Comment:
This would only work if we compile with RtAudio. Let's put this outside of the #ifdef or inside setSampleRate() and setAudioBufferSizeISamples().

Reply:
I didn't do that because I think the GUI components to set these values inside QJackTrip only work when you are using RtAudio.

Reply:
Oh, I thought that's what this change is about. The Flatpak only builds the JACK backend. So for this we need to change the GUI, too.

So you are testing this with the ALSA backend of RtAudio right now?

Reply:
Yes, I tested both ALSA and JACK in both Classic and VS mode. The limitation here is that you can only CHANGE settings in Classic if you are using RtAudio, but JACK will still work. Both work in VS mode.